### ISSUE TITLE
Sound control should move to "Mail Rules"

### ISSUE THREAD

**Issue by IvanKlimovich** [nylas/N1#2577]
Hello N1 team. Thanks a lot for your product.  N1 has become the first Desktop client which I can use without feeling pain. Except one small thing: Sound. 

I get close to 200 emails every day, from 4 different accounts. I have some subscribtions, which constantly send me new information, but which are automatically marked as "Read" for future consumption etc. use cases are unlimited 

Out of those 200 emails, only 10-30 deserve sound notification. It would be perfect if I could have sound notifications only for those ( this way if I hear sound, I know this is super important)

I think that the easiest way to implement this is by adding sound option to the rules menu:
![n1 sounds](https://cloud.githubusercontent.com/assets/6802057/16661057/c2eb5d62-4479-11e6-952b-107f68ce0568.JPG)

The only replated issue, I could find: #788 . Choosing the sound would be also a very good idea.

**Comment by jackiehluo** [comment]
Hm, would that be similar to #2356?

**Comment by IvanKlimovich** [comment]
Yes, it is very close. Although, my proposal is a bit more flexible.

**Comment by dcrystalj** [comment]
Now its also impossible to disable sound for one mail account on not for the other :-1:
